guides/common/modules/con_managing-alternate-content-sources.adoc

```diff
@@ -2,9 +2,43 @@
 = Managing Alternate Content Sources
 
 Alternate content sources define alternate paths to download content during synchronization.
-The content itself is downloaded from the alternate content source and only the metadata is downloaded from the upstream URL.
+The content itself is downloaded from the alternate content source, while the metadata is downloaded from the {ProjectServer} or the upstream URL, depending on the configuration.
 You can use alternate content source to speed up synchronization if the content is located on the local filesystem or on a nearby network.
 You can set up alternate content sources for {ProjectServer} and {SmartProxy}.
 You must refresh the alternate content source after creation or after making any changes.
 A weekly cron job refreshes all alternate content sources.
 You can also refresh the alternate content sources manually using the {ProjectWebUI} or the Hammer CLI.
+
+There are three types of alternate content sources:
+
+Custom::
+Custom alternate content sources download the content from any upstream repository on the network or filesystem.
+
+Simplified::
+Simplified alternate content sources copy the upstream repository information from your {ProjectServer} for the selected products.
+Simplified alternate content sources are ideal for situations where the connection from your {SmartProxy} to the upstream repo is faster than from your {ProjectServer}.
+ifdef::satellite[]
+Selecting the Red Hat products when creating a simplified alternate content source will download the content to the {SmartProxies} from the {Team} CDN.
+endif::[]
+
+RHUI::
+RHUI alternate content sources download content from a Red Hat Update Infrastructure server.
+ifdef::satellite[]
+The RHUI alternate content source must be RHUI version 4 or greater and use the default installation configuration. For example, AWS RHUI is unsupported because it uses an installation scenario with unique authentication requirements.
```

Comment:
```suggestion
The RHUI alternate content source must be RHUI version 4 or greater and use the default installation configuration.
For example, AWS RHUI is unsupported because it uses an installation scenario with unique authentication requirements.
```